plugin/README.md

```diff
@@ -22,6 +22,10 @@ Plugin inputs can be set in `netlify.toml`. They're all optional.
   - `feat/*`: matches all branches starting with `feat/`
   - `*-bug`: matches all branches finishing with `-bug`
 - `disabled` - _Default: `false`_ - Use to disable the plugin without removing it.
+- `pathPrefix` - The prefix of your website if it's not at the root level.
```

Comment:
It's not super clear to me. Can you explain concretely the situation and what it this option does? (Netlify website hosted on `/pathPrefix` in production, URLs hardcoded in sitemaps, the option is removing the `pathPrefix` from all urls found on the website)

Reply:
isn't the sentence just below enough?

Reply:
As a user discovering the doc I'm not sure I would understand why I need it.
Like: "how can my website not be at the root level, it is hosted on `xxx.netlify.app/`!"
But ok, they can try without and they will see if they need it.

Reply:
I see what you mean, but as we aim to be zero config I think most people will end up here with a problem to solve not just reading to discover. Happy to improve over time though